FSI/Force.C

```diff
@@ -167,7 +178,7 @@ Foam::tmp<Foam::volScalarField> preciceAdapter::FSI::Force::mu() const
     }
 }
 
-void preciceAdapter::FSI::Force::write(double * buffer, bool meshConnectivity, const unsigned int dim)
+void preciceAdapter::FSI::Force::write(double * buffer, bool /*meshConnectivity*/, const unsigned int dim)
```

Comment:
Why commenting the names out?

Reply:
Depending on your compiler configuration, this might lead to an unused warning. Wmake doesn't complain,  I will revert it.

Reply:
WMake still triggers the same compiler and there should be a warning (unless explicitly disabled).

The warning gives us a signal that we maybe need to change something in the design: `meshConnectivity` seems to be "a different kind of animal" than `buffer` and `dim`, which we explicitly need to know for writing. So, we should either overload `write` to have one version with and one without `meshConnectivity`, or select a different one using a [strategy pattern](https://sourcemaking.com/design_patterns/strategy). Or maybe move `meshConnectivity` to some kind of "repository" (if I remember correctly, that's the "repository pattern"), so that we don't need to propagate it all the way.

Reply:
The current implementation is consistent with the other modules. Hence, these changes would affect the whole adapter and require a separate PR. However, (what I meant and it is just a guess) configuring the code with e.g. cmake would probably result in a compiler warning, whereas wmake doesn't trigger any warning. 
Removing/Commenting the variable is in any case a clean way to avoid warnings.

Reply:
It would definitely be a bigger work package, yes. Let's open an issue for this before resolving this conversation.

Regarding the specific PR: I am fine doing this, as long as there is a comment next to it explaining why we do it.

Reply:
Alright, keep in mind that we call this methods unified through the CouplingDataUser and in some other functions, the `meshConnectivity` is needed. So, all arguments are passed in to the function. This was also the reason, why I added them here, although they are not needed. I am not familiar with pattern methods. Do you open up the issue? Then, you could maybe directly describe, what you have in mind.